service/internal/auth/config.go

```diff
@@ -4,12 +4,12 @@ import "fmt"
 
 // AuthConfig pulls AuthN and AuthZ together
 type Config struct {
-	Enabled     bool `yaml:"enabled" default:"true" `
```

Comment:
Suggestion: Rename field to `DeprecatedEnabled` and print a warning or error out if it is set to `false`. This will help people transition to the new config format

Reply:
I like it! I'll do that. thanks!